src/pepr/uds-operator-config/values.yaml

```
@@ -6,4 +6,4 @@ operator:
   UDS_LOG_LEVEL: "###ZARF_VAR_UDS_LOG_LEVEL###"
   AUTHSERVICE_REDIS_URI: "###ZARF_VAR_AUTHSERVICE_REDIS_URI###"
   # Allow Pepr watch to be configurable to react to dropped connections faster
-  PEPR_LAST_SEEN_LIMIT_SECONDS: "300"
+  PEPR_LAST_SEEN_LIMIT_SECONDS: "60"
```

In testing this actually hasn't yielded much better results. We may want to also include this override:
```suggestion
  PEPR_LAST_SEEN_LIMIT_SECONDS: "60"
  PEPR_RELIST_INTERVAL_SECONDS: "60"
```

This should effectively make pepr poll every 60s with a GET call on the k8s api.

---

Added - thanks!